auxiliary.py

```diff
@@ -5,11 +5,11 @@
 
 
 uco = mpatches.Patch(color=(1., 1., 0.), label='UCO')
-buffer = mpatches.Patch(color=(1., 0.5, 0.), label='Buffer')
+buffer = mpatches.Patch(color=(1., 0.5, 0.), label='Buffer layer')
 pyc = mpatches.Patch(color=(1., 0., 0.), label='PyC')
 sic = mpatches.Patch(color=(0.6, 0.3, 0.), label='SiC')
-matrix = mpatches.Patch(color=(0.63, 0.63, 0.63), label='Matrix')
-block = mpatches.Patch(color=(0.61, 1., 0.91), label='Block')
+matrix = mpatches.Patch(color=(0.63, 0.63, 0.63), label='Compact matrix')
```

Comment:
This is still unclear to me. What material is it made of? Compact and matrix are both geometric descriptions, but the labels should ideally convey what materials (not what shapes).